client/src/app/site/services/reroute.service.ts

```diff
@@ -63,16 +65,16 @@ export class RerouteService {
     /**
      * If the user requested a route without direct meaning, forward to their meaningful meeting
      */
-    public forwardToOnlyMeeting(segments: string[] = []): void {
+    public forwardToOnlyMeeting(segments: string[] = []): UrlTree | void {
```

Comment:
With your changes to take out the navigation most of the `RerouteService`'s method names don't fit anymore, please change names like `forwardToOnlyMeeting` that insinuate that they also handle the navigation to something more fitting like maybe `getOnlyMeetingUrlTree` (please don't forget to change the documentation either)